libs/sdk-ui-kit/styles/scss/createThemeDialog.scss

```diff
@@ -0,0 +1,218 @@
+// (C) 2022 GoodData Corporation
+.gd-styling-editor-dialog {
+    font-family: gdcustomfont, avenir, "Helvetica Neue", arial, sans-serif;
+    max-height: 560px;
+    width: 100vw;
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+
+    &-create {
+        max-width: 830px !important;
+    }
+
+    &-edit {
+        max-width: 600px !important;
+    }
+
+    &-header {
+        background-color: var(--gd-palette-complementary-0, white);
```

Comment:
If this var definition already exists as a scss variable, please use it instead of defining it again.